src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((declaration) => declaration.trim().length > 0)
+    .map((declaration) => {
+      const [property, value] = declaration
+        .split(':')
+        .map((item) => item.trim());
+
+      // Залишити властивість з дефісами
+      return { [property]: value };
```

Comment:
The map function currently returns an object for each CSS declaration, which is not necessary. Instead, you can directly return an array of arrays or tuples, which can then be converted into an object using reduce.